### PR TITLE
added clarity in studio

### PR DESCRIPTION
**Pull request by muhammadali124**
**Description:** 
This PR adds clarity events in course list and create new course page in studio.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-6443

### CODE REVIEW

**Review by mfarhan943 — APPROVED** [review]

File: cms/envs/production.py [hunk 1]
```diff
@@ -600,3 +600,9 @@ def get_env_setting(setting):
     'USETIFUL_PROJECT_TOKEN',
     None
 )
+
+# MS CLARITY PROJECT ID
+CLARITY_PROJECT_ID = ENV_TOKENS.get(
+    'CLARITY_PROJECT_ID',
+    'replace-me'
```
Comment (Anas-hameed) [inline cms/envs/production.py:607]: Can we have real value(project id) fallback here.

**Comment by mfarhan943** [comment]
The description needs to be improved.

File: cms/envs/production.py (hunk 1 above)
Comment (taimoor-ahmed-1) [inline cms/envs/production.py:607]: CLARITY_PROJECT_ID = ENV_TOKENS.get(
    'CLARITY_PROJECT_ID',
    CLARITY_PROJECT_ID

Comment (muhammadali124) [reply cms/envs/production.py]: done

**Pull request merged**